src/elm/Page/Community/Settings/Features.elm

```diff
@@ -196,6 +251,11 @@ update msg model loggedIn =
                 |> UR.init
                 |> saveFeaturePort loggedIn Objectives model.status state
 
+        ToggleKyc state ->
+            { model | hasKyc = state }
+                |> UR.init
+                |> saveFeaturePort loggedIn Kyc model.status state
```

Comment:
```suggestion
        ToggleKyc state ->
            { model | hasKyc = model.hasKyc }
                |> UR.init
```

Here we changed ToggleKyc to always use the initial state, so it doesn't change when clicked. We should also probably remove the saveFeaturePort, so it won't send a useless request to the backend :)

Reply:
Awesome! Thanks Rodrigo!!